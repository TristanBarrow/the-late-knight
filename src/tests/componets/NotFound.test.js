import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../componets/NotFound';

test('render 404 not found page', () => {
  const wrapper = shallow(<NotFound />);
  expect(wrapper).toMatchSnapshot();
});