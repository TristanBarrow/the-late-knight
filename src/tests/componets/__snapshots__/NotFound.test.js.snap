// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 404 not found page 1`] = `
<div>
  404!!! - 
  <Link
    replace={false}
    to="/"
  >
    Go to Home Page
  </Link>
</div>
`;
